packages/trpc/server/admin-router/router.ts

```diff
@@ -118,4 +121,20 @@ export const adminRouter = router({
       });
     }
   }),
+  deleteDocument: adminProcedure
+    .input(ZAdminDeleteDocumentMutationSchema)
+    .mutation(async ({ input }) => {
+      const { id, userId, reason } = input;
+      try {
+        await deleteDocument({ id, userId });
+        await sendDeleteEmail({ documentId: id, reason });
+        return;
```

Comment:
We might make this a `{ success: true }` kind of return, while we can have tRPC return `undefined` as it would here we should provide something a little more enhanced.